class ApplicationError(Exception):
    pass


class ArgumentError(Exception):
    pass
